### PR TITLE
Pandaria fashion show

### PR DESCRIPTION
**Pull request by sejtalk**
## Changelog:
- Added Yaungol clothes.
- Added Hozen clothes.
- Added Mantid clothes.

# How to test:
Check clothes for Yaungol, Hozen and Mantid characters.

### CODE REVIEW

File: interface/portraits/portraits_hozen_sprites.gfx [hunk 1]
```diff
@@ -26,13 +26,19 @@ spriteTypes = {
...
 		name = "GFX_hozen_male_neck"
 		texturefile = "gfx\\characters\\hozen_male\\hozen_male_neck_1.dds"
 		noOfFrames = 7
 		norefcount = yes
 	}
+	
+	spriteType = {
+		name = "GFX_hozen_male_clothes"
+		texturefile = "gfx\\characters\\hozen_male\\hozen_male_clothes.dds"
+		noOfFrames = 11
```
Comment (zumbak04) [inline interface/portraits/portraits_hozen_sprites.gfx:39]: There should be 12 slots.

File: interface/portraits/portraits_yaungol_sprites.gfx [hunk 2]
```diff
@@ -89,5 +89,12 @@ spriteTypes = {
 		noOfFrames = 3
 		norefcount = yes
 	}
+	
+	spriteType = {
+		name = "GFX_yaungol_male_clothes"
+		texturefile = "gfx\\characters\\yaungol_male\\yaungol_male_clothes.dds"
+		noOfFrames = 11
```
Comment (zumbak04) [inline interface/portraits/portraits_yaungol_sprites.gfx:96]: There should be 12 slots.

**Comment by sejtalk** [comment]
> Some comments:
> 
> * Yaungol clothes 5-7 are all the same. I believe 10 is also the same as 5-7. Clothes do not appear to be implemented for female Yaungol.
> * Mantid clothes 5-7 are all the same. I believe 10 is also the same as 5-7. Clothes do not appear to be implemented for female mantid.
> * Hozen clothes 3-4 are duplicates. 10-11 are also duplicates.
> 
> Otherwise, everything looks good. I particularly like the hozen clothes, nice job!

Fixed the bug where clothes had only 11 slots so im asking for a re-review

**Review by Myrran — COMMENTED** [review]
Looks fine except for this:

- Yaungol male clothes disappear when choosing middleaged or old age in ruler designer
- Yaungol female clothes are missing
- Mantid female clothes are missing

**Comment by sejtalk** [comment]
> Looks fine except for this:
> 
> * Yaungol male clothes disappear when choosing middleaged or old age in ruler designer
> * Yaungol female clothes are missing
> * Mantid female clothes are missing

Yaungol female use tauren portraits and mantid females dont have clothes in WoW. Will fix Yaungol male clothes asap

**Review by Myrran — APPROVED** [review]
Yaungol male clothes work fine.

**Comment by zumbak04** [comment]
Yaungol can't change beard.
<details>
<summary>Click to expand</summary>

image

</details>
It's because of GFX_yaungol_male_fur. It should be named `GFX_yaungol_male_beard_behind`.

**Comment by zumbak04** [comment]
Hozen and yaungol LGTM, but I demand mantid headgears and clothes of different colours.

**Review by zumbak04 — COMMENTED** [review]
🔼

**Comment by sejtalk** [comment]
> Yaungol can't change beard.
> 
> Click to expand
> It's because of `GFX_yaungol_male_fur`. It should be named `GFX_yaungol_male_beard_behind`.

fixed

**Comment by sejtalk** [comment]
> Hozen and yaungol LGTM, but I demand mantid headgears and clothes of different colours.

Mantid have different clothes and headgear for emperor tier, king tier and duke tier, intrigue clothes and martial clothes. I don't get the problem

**Comment by zumbak04** [comment]
> > Hozen and yaungol LGTM, but I demand mantid headgears and clothes of different colours.
> 
> Mantid have different clothes and headgear for emperor tier, king tier and duke tier, intrigue clothes and martial clothes. I don't get the problem

They're not different.
These 2 are the same.
<details>
<summary>Click to expand</summary>

image

</details>
These 2 too.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by sejtalk** [comment]
> > > Hozen and yaungol LGTM, but I demand mantid headgears and clothes of different colours.
> > 
> > 
> > Mantid have different clothes and headgear for emperor tier, king tier and duke tier, intrigue clothes and martial clothes. I don't get the problem
> 
> They're not different.
> These 2 are the same.
> 
> Click to expand
> image
> 
> These 2 too.
> Click to expand
> image

yeah but the headgear/cloth mix is different

**Comment by zumbak04** [comment]
> yeah but the headgear/cloth mix is different

Em, wha? Are you going to replace duplicates or not?

**Comment by zumbak04** [comment]
They may be different colours too.
<details>
<summary>Click to expand</summary>

image

</details>

**Comment by sejtalk** [comment]
> > yeah but the headgear/cloth mix is different
> 
> Em, wha? Are you going to replace duplicates or not?

Did it

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Pull request merged**